prisma/schema.prisma

```diff
@@ -147,24 +148,45 @@ enum MembershipRole {
     VIEWER
 }
 
-model Trace {
+model TraceSession {
```

Comment:
I assume you did not want to take `Session` as this was taken above already?

Reply:
yep, resolve clash with auth session, on all apis/sdks it will only be `session`